README.md

```diff
@@ -87,22 +107,18 @@ Customize in `~/.laptop.local`
 
 Your `~/.laptop.local` is run at the end of the Laptop script.
 Put your customizations there.
-For example:
+Possible customizations are:
 
 ```sh
 #!/bin/sh
 
-brew_tap 'caskroom/cask'
-brew_install_or_upgrade 'brew-cask'
+brew_install_or_upgrade 'mongodb'
+brew_launchctl_restart 'mongodb'
```

Comment:
Do these functions actually work when placed in `~/.laptop.local`? I have a feeling it's not run in the context of the main laptop script (and thus wouldn't have access to functions defined there).